### PR TITLE
LPS-77063 Add condition to get dropdown node

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test

**Comment by rafaprax** [comment]
ci:test

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy [hunk 1]
```diff
@@ -48,7 +48,7 @@
 						{param strings: $strings /}
 					{/call}
 				{/if}
-				{if $fixedOptions}
+				{if $fixedOptions and (length($fixedOptions) > 0)}
 					{if not $readOnly and $displayValue}
```
Comment (clovisdasilvaneto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy]: Hey @alinedoleron, just for a better legibility, can you agrupe this condition with parentheses?

Like: {if ($fixedOptions) and (length($fixedOptions) > 0)}

**Comment by clovisdasilvaneto** [comment]
Hey @alinedoleron, I'll ask with Front End Guild guys if they can take a look in this PR, Ok?

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy (hunk 1 above)
Comment (clovisdasilvaneto) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy]: @interaminense Cay you take a look in this issue?

**Comment by clovisdasilvaneto** [comment]
Pull request forwarded to https://github.com/pedroqueiroz94/liferay-portal/pull/80. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**